scripts/cmake/Irep2Optimization.cmake

```diff
@@ -2,10 +2,8 @@
 # This is required by MSVC
 
 if(CMAKE_CXX_COMPILER_ID STREQUAL "GNU")
-    if (CMAKE_CXX_COMPILER_VERSION VERSION_LESS "10.0.0")
```

Comment:
Is this a problem on newer versions as well?

Reply:
Yeah, the preprocessed headers are needed more because of MSVC limits  rather than optimisation. It's fine to skip them.